File: NpowerN.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
  long Number, Counter = 1, Multiple = 1;
  Scanner scanner = new Scanner(System.in);
  System.out.print("Enter a number to multiply the number itself is the number of times:");
  Number = scanner.nextInt();
  while(Counter <= Number) {
    Multiple = Multiple * Number;
    Counter++;
  }
  System.out.print(Number + " multiplied " + Number + " times is " + Multiple + ".");
  }
}

File: javascript2/NpowerN.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
 var Counter = 1
var Multiple = 1
print("Enter a number to multiply the number itself is the number of times:");
var Count = parseInt(readln())
while(Counter <= Count) {
  Multiple = Multiple * parseInt(Count)
  Counter = Counter + 1
}
print(Count + " multiplied " + Count + " times is " + Multiple + ".")